example/parse_multiple_files/main.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"log"
+	"os"
+
+	"github.com/ilyakaznacheev/cleanenv"
+)
+
+type config struct {
+	Port         string         `yaml:"port"`
+	JWTSecret    string         `yaml:"jwt_secret"`
+	Database     databaseConfig `yaml:"database"`
+	EmailService emailService   `yaml:"email_service"`
+}
+
+type databaseConfig struct {
+	User     string `yaml:"user"`
+	Password string `yaml:"password"`
+	Name     string `yaml:"name"`
+	Host     string `yaml:"host"`
+	Port     string `yaml:"port"`
+	SSLMode  string `yaml:"ssl_mode"`
+}
+
+type emailService struct {
+	EmailKey                    string `yaml:"email_key"`
+	AccountValidationTemplateID string `yaml:"account_validation_template_id"`
+}
+
+func main() {
+	if len(os.Args[1:]) == 0 {
+		log.Println("Provide a list of yaml files for configuration")
+		return
+	}
+
+	cfg, err := ParseConfigFiles(os.Args[1:]...)
```

Comment:
Looks great, but since this is an example, it may be easier to understand, if you will hardcode these two file path instead of parsing free input from CLI parameters.

Or at least add a comment here, that in our example these files will be `db_config.yaml` and `general_config.yaml`.